rust/cardano-blockchain-types/src/stake_address.rs

```diff
@@ -0,0 +1,235 @@
+//! A stake address.
+
+// cspell: words Scripthash, Keyhash
+
+use std::fmt::{Display, Formatter};
+
+use anyhow::{anyhow, Context};
+use pallas::{
+    crypto::hash::Hash,
+    ledger::{
+        addresses::{
+            ShelleyAddress, ShelleyDelegationPart, ShelleyPaymentPart,
+            StakeAddress as PallasStakeAddress,
+        },
+        primitives::conway,
+    },
+};
+
+use crate::Network;
+
+/// A stake address.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Hash)]
+pub struct StakeAddress(PallasStakeAddress);
+
+impl StakeAddress {
+    /// Creates a new instance from the given parameters.
+    // The panic can only occur if the implementation is incorrect.
+    #[allow(clippy::expect_used, clippy::missing_panics_doc)]
```

Comment:
As we are already have a comment related to panic, we can add a doc Panics section and remove `clippy::missing_panics_doc`

Reply:
Discussed: I have removed unnecessary comment.